components/grid/compact-grid.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Grid Size - Compact Grid
+page_title: Grid Size | Grid for Blazor
+description: Adjust the size and modify the appearance of the Telerik UI for Blazor Grid.
+slug: compact-grid
+tags: telerik,blazor,grid,size,compact
+published: true
+position: 20
+---
+
+## Grid Size
+
+This feature attempts to address the need for a **Compact Grid**, which renders more items by utilizing the available space, mainly through setting smaller padding in its cells.
+
+To utilize this feature, set the [`Size`](#grid-size) property of the component. It accepts values of type [`ThemeConstants.Grid.Size`](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.ThemeConstants.Grid.Size) and the supported values are: 
+
+1. `Small` - Sets the padding of the building blocks of the Grid to **2px 8px**.
```

Comment:
Let's refactor that section to list the size options in a table. 

I also suggest omitting the specific padding values - these may be changed in some future version of the themes, so it will be hard to maintain. Plus, you'we already listed that the setting focuses on setting smaller paddings.